### PR TITLE
Shin/ch1264/sync hw connect dialog with new proposal

### PR DESCRIPTION
**Pull request by sarveshwar-singh**
### After Change
image

image

### This PR also fixes

- [X] https://github.com/Emurgo/yoroi-frontend/pull/549/commits/ef1b38fafa668a971819070de4fcdd931710efa5 manage-translations

- [X] https://github.com/Emurgo/yoroi-frontend/pull/549/commits/eea640e8a3a4e4e26793cedbce039e879e838721 increase close icon size on dialogs 

- [X] https://github.com/Emurgo/yoroi-frontend/pull/549/commits/3a4af6c948259cc5c072c017ea4b1e5d702d6aee removed un-wanted routes

### CODE REVIEW

File: app/i18n/locales/en-US.json [hunk 1]
```diff
@@ -277,18 +272,19 @@
...
   "wallet.create.dialog.repeatPasswordLabel": "Repeat spending password",
   "wallet.create.dialog.title": "Create wallet",
-  "wallet.create.type.mnemonic.title": "Wallet from 15 mnemonic words",
-  "wallet.create.type.mnemonic.description": "The simplest and most common way to create a Wallet. Yoroi will generate 15 mnemonic words that you will have to store in a safe place in order to restore the wallet.",
+  "wallet.create.dialog.walletNameHint": "e.g: Shopping Wallet",
+  "wallet.create.dialog.walletPasswordLabel": "Spending password",
+  "wallet.create.tabs.advanced.title": "Advanced",
+  "wallet.create.tabs.recommended.title": "Recommended",
+  "wallet.create.type.business.title": "Business Wallet",
+  "wallet.create.type.ledger.description": "!!!Ledger Hardware Wallet Descriprion",
```
Comment (SebastienGllmt) [inline app/i18n/locales/en-US.json]: EN-US file should not have any `!!!` in it

File: app/i18n/locales/en-US.json [hunk 2]
```diff
@@ -277,18 +272,19 @@
...
+  "wallet.create.type.ledger.title": "Ledger Hardware Wallet",
   "wallet.create.type.master.title": "Master Key",
+  "wallet.create.type.mnemonic.description": "The simplest and most common way to create a Wallet. Yoroi will generate 15 mnemonic words that you will have to store in a safe place in order to restore the wallet.",
+  "wallet.create.type.mnemonic.title": "Wallet from 15 mnemonic words",
   "wallet.create.type.paper.title": "Paper Wallet",
-  "wallet.create.type.business.title": "Business Wallet",
+  "wallet.create.type.trezor.description": "!!!Trezor Hardware Wallet Descriprion",
```
Comment (SebastienGllmt) [inline app/i18n/locales/en-US.json]: EN-US file should not have any `!!!` in it

**Comment by rnd-at-emurgo** [comment]
<details>
<details>
  <summary>Complete E2E screenshots collection for 'PR549-725955e'</summary>



</details>

**Comment by rnd-at-emurgo** [comment]
<details>
<details>
  <summary>Complete E2E screenshots collection for 'PR549-ff27724'</summary>



</details>

**Pull request merged**